modules/selectors.js

```diff
@@ -0,0 +1,17 @@
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+// const listSec = document.getElementById('list-sec');
+// const addSec = document.getElementById('add');
+// const contactSec = document.getElementById('contact');
```

Comment:
- [OPTIONAL] I suggest removing unused code from here to make your code clean and to Follow the [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) 👌💯

![image](https://user-images.githubusercontent.com/45231276/179382779-204565b5-b55f-4edd-8b14-eb0af2fdcf57.png)

Reply:
Thanks!